drivers/usbc/tcpc/ucpd_stm32gx_priv.h

```diff
@@ -0,0 +1,309 @@
+/*
+ * Copyright (c) 2022  The Chromium OS Authors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_USBC_DEVICE_UCPD_STM32X_PRIV_H_
+#define ZEPHYR_DRIVERS_USBC_DEVICE_UCPD_STM32X_PRIV_H_
+
+#include <drivers/usbc/usbc_tcpc.h>
+#include <stm32_ll_ucpd.h>
```

Comment:
nit: headers should be self-contained: e.g. misses `sys/util_macro.h`

Reply:
I added #include <sys/util.h>